src/pages/landing/ebooks/components/_get-ebook.tsx

```diff
@@ -1,6 +1,7 @@
 import React from 'react'
 import styled from 'styled-components'
 import Cookies from 'js-cookie'
+import { Link } from "gatsby";
```

Comment:
```suggestion
import Link from 'features/components/atoms/link'
import useRegion from 'components/hooks/use-region'
```

Reply:
Let's import new `Link` rebranding component which contains internal PDF links logic and `useRegion` to have different links for EU and nonEU PDF.